client/my-sites/site-monitoring/style.scss

```
@@ -0,0 +1,32 @@
+@import "uplot/dist/uPlot.min.css";
+
+.u-legend {
+	color: var(--studio-gray-100);
+
+	// Disable rendering first u-marker, which is for "Date" and is empty.
+	> .u-series:first-child > th > .u-marker {
+		display: none;
+	}
+
+	// Use increased specificity to override original uPlot styles.
+	> tr.u-series > th {
+		cursor: initial;
+
+		&::after {
+			vertical-align: baseline;
+		}
+		> .u-marker {
+			// Nudge marker down to align with text.
+			margin-top: -2px;
+			// Make the marker round.
+			border-radius: 100%;
+		}
+		> .u-label {
+			vertical-align: baseline;
+		}
+	}
+}
+
+.uplot {
+	border: 1px solid #dcdcde;
```

Instead of direct hex colors, it's better to use variables. Although those variables may not work in a storybook.
```suggestion
	border: 1px solid var(--studio-gray-5);
```